src/Stoa.ts

```diff
@@ -252,6 +252,11 @@ class Stoa extends WebService {
             isBlackList,
             this.getWalletTransactionsHistory.bind(this)
         );
+        this.app.get(
+            "/wallet/transaction/history/:address",
+            isBlackList,
+            this.getWalletTransactionHistory.bind(this)
+        );
```

Comment:
/wallet/transactions/history/:address <- Are you going to remove this endpoint later?
If so, it's good to write a comment to remove it later.

Reply:
I added `@deprecated`